### PR TITLE
Ft expand

### PR DESCRIPTION
**Pull request by j042**

### CODE REVIEW

File: gustaf/create/faces.py [hunk 1]
```diff
@@ -203,3 +203,190 @@ def to_quad(tri):
...
+    quad[:, 2] = edges.edges[:, 1] + size
+    quad[:, 3] = edges.edges[:, 0] + size
+
+    return Faces(vertices, quad)
+
+
+def _two_ortho(trajectory):
+    """
+    Finds two orthogonal vectors to trajectory.
+    This won't work if the vector is multiples of [-1, 1, -1].
```
Comment (jzwar) [inline gustaf/create/faces.py:251]: 🤣

**Review by jzwar — APPROVED** [review]
Love the new features. Please bump version so a new wheel will be available shortly.

LGTM, thanks a lot!

**Comment by jzwar** [comment]
Please merge once done

**Comment by j042** [comment]
> Please merge once done

P >> 1

**Comment by j042** [comment]
Screenshot 2024-02-29 at 20 38 37

**Comment by j042** [comment]
Screenshot 2024-02-29 at 20 41 19

File: gustaf/utils/arr.py [hunk 2]
```diff
@@ -525,3 +525,83 @@ def is_one_of_shapes(arr, shapes, strict=False):
...
+        normals = np.empty_like(der)
+        normals[:, 0] = der[:, 1]
+        normals[:, 1] = -der[:, 0]
+
+    elif shape[2] == 3:
+        der = derivatives.reshape(shape[0] * shape[1], shape[2])
+        normals = cross3d(der[::2], der[1::2])
+
+    if normalize:
+        normals *= 1.0 / np.linalg.norm(normals, axis=1).reshape(-1, 1)
```
Comment (j042) [inline gustaf/utils/arr.py:561]:
```suggestion
        normals /= np.linalg.norm(normals, axis=1).reshape(-1, 1)
```

Comment (jzwar) [reply gustaf/utils/arr.py:561]: nice, makes it inplace

**Pull request merged**